collections_.py

```diff
@@ -301,14 +301,16 @@ def get(self, cid):
                 {'$group': {'_id': '$session'}},
                 ])
         query = {'_id': {'$in': [ar['_id'] for ar in agg_res]}}
-        projection = {'label': 1, 'subject.code': 1, 'notes': 1, 'timestamp': 1, 'timezone': 1}
-        projection['permissions'] = {'$elemMatch': {'_id': self.uid, 'site': self.source_site}}
-        sessions = list(self.dbc.find(query, projection))
+        projection = {'label': 1, 'subject.code': 1, 'notes': 1, 'project': 1, 'group': 1, 'timestamp': 1, 'timezone': 1}
+        sessions = self._get(query, projection, self.request.get('admin').lower() in ('1', 'true'))
```

Comment:
We actually don't want to use ```ContainerList._get()``` in order to avoid the permissions check, as the user does not actually need permissions on the session in order to get to the acquisitions. We also don't want to expose the project, as done below. This will need to be addressed in FDM.
TODO: specify permission semantics, especially wrt collections.